Brisk/AppDelegate.swift

```diff
@@ -70,4 +73,28 @@ final class AppDelegate: NSObject, NSApplicationDelegate {
         self.statusItem?.remove()
         self.statusItem = nil
     }
+
+    private func registerRadarURLs() {
+        let eventManager = NSAppleEventManager.shared()
+        let selector = #selector(handleGetURLEvent(event:withReplyEvent:))
+        eventManager.setEventHandler(self, andSelector: selector,
+                                     forEventClass: AEEventClass(kInternetEventClass),
+                                     andEventID: AEEventID(kAEGetURL))
+    }
+
+    @objc
+    private func handleGetURLEvent(event: NSAppleEventDescriptor, withReplyEvent: NSAppleEventDescriptor) {
+        let radarID = event.paramDescriptor(forKeyword: keyDirectObject)?.stringValue.flatMap(radarID(from:))
+        if let radarID = radarID {
+            _ = self.dupeRadarMenuItem.target?.perform(self.dupeRadarMenuItem.action!)
```

Comment:
I would force unwrap target here, this is a programming error if it isn't connected